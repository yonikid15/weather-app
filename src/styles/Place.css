.place {
  grid-column: span 3;
  grid-row: 1;
  display: flex;
  flex-direction: column;

  &__city {
    font-size: 35px;
    font-weight: 400;
    margin: 0;
    padding: 0;
  }

  &__country {
    font-size: 18px;
    font-weight: 100;
    margin-top: 5px;
    margin-left: 35px;
    padding: 0;
  }

  &__icon {
    font-size: 25px;
  }
}