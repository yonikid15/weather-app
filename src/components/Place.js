import React from 'react';
import { countries } from 'country-data';
import styles from '../styles/Place.css';

const Place = ({ city, country }) => {
  return (
    <div className={ styles.place }> 
      <h3 className={ styles.place__city }>
        <span>{city}</span>
      </h3>
      <h5 className={ styles.place__country }>
        { countries[ country ].name }
      </h5>
    </div>
  );
};

export default Place;