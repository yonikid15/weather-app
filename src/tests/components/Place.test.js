import React from 'react';
import { shallow } from 'enzyme';
import Place from '../../components/Place';

test('should render Place correctly', () => {
  const wrapper = shallow( <Place city={'Ottawa'} country={'CA'} /> );
  expect( wrapper ).toMatchSnapshot();
});