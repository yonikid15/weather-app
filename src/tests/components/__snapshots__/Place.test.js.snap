// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Place correctly 1`] = `
<div
  className="place"
>
  <h3
    className="place__city"
  >
    <span>
      Ottawa
    </span>
  </h3>
  <h5
    className="place__country"
  >
    Canada
  </h5>
</div>
`;
